submissions/chiconnect-mobile-money-payout/src/App.jsx

```diff
@@ -0,0 +1,184 @@
+        <h1 className='my-5 mx-auto text-2xl font-semibold text-slate-700'>
+          Pay with Mobile Money
+        </h1>
+
+        <span className='block text-md font-medium text-slate-700'>
+          Momo code
+        </span>
+
+        <select name='momoCode' placeholder='MPS'
+          value={paymentData.momoCode} onChange={handleFormChange}
+          className='mt-1 px-3 py-2 bg-white border shadow-sm 
+            border-slate-300 focus:outline-none focus:border-purple-500
+            w-full rounded-md sm:text-sm focus:ring-1'>
+          {
+            momoCodes?.data?.map((momoCode) => (
+              <option
+                key={`${momoCode.name}-${momoCode.code}`}
+                value={momoCode.code}
+                onClick={() => setPaymentData(prevData => (
+                  {
+                    ...prevData,
+                    country: momoCode.country
+                  }
+                ))}>
+                {`${momoCode.code} — ${momoCode.name}`}
+              </option>
+            ))
+          }
+        </select>
+
+        <div className='w-full flex flex-row mt-6 justify-between items-center'>
+          <div className='pr-4'>
+            <span className='block text-md font-medium text-slate-700'>
+              Phone number
+            </span>
+
+            <input name='phoneNumber' placeholder='+234912345678'
+              value={paymentData.phoneNumber}
+              onChange={handleFormChange}
+              className='mt-1 px-3 py-2 bg-white border shadow-sm 
+          border-slate-300 focus:outline-none focus:border-purple-500
+          w-full rounded-md sm:text-sm focus:ring-1' />
+          </div>
+
+          <div className='pl-4'>
+            <span className='block text-md font-medium text-slate-700'>
+              Amount (USD)
+            </span>
+
+            <input name='amount' placeholder='10'
+              value={paymentData.amount} onChange={handleFormChange}
+              className='mt-1 px-3 py-2 bg-white border shadow-sm 
+            border-slate-300 focus:outline-none focus:border-purple-500
+             w-full rounded-md sm:text-sm focus:ring-1' />
+          </div>
+        </div>
+
+        <div className='w-full flex flex-col justify-center items-center'>
+          {
+            error.length > 0 &&
+            <span className='text-red-500 text-sm font-semibold mt-8'>
+              {error}
+            </span>
+          }
+
+          {
+            info.length > 0 &&
+            <span className='text-purple-500 text-center max-w-md text-sm font-semibold mt-8'>
+              {info}
+            </span>
+          }
+
+          <button onClick={handlePayClick}
+            className={`${error.length === 0 ? 'mt-8' : 'mt-2'} px-12 py-2 border shadow-sm text-white rounded-xl 
+            font-semibold bg-purple-500 hover:border-purple-500 tracking-wider
+            hover:bg-purple-50 hover:text-purple-500 hover:scale-95 transition-all`}>
+            PAY NOW
```

Comment:
Are they paying? Or Sending, Transferring, Gifting or something else?

Reply:
@uchibeke  Please sir, I'm not fully sure I get this part. But I think it could depend on the user.

Reply:
They are paying